### PR TITLE
Add Stormer-Verlet notebook.

### PR DESCRIPTION
**Pull request by ketch**
This adds a notebook that includes the calculations relevant to Stormer-Verlet in the paper.  See https://github.com/ketch/2021_bseries_paper/pull/31/.

### CODE REVIEW

File: notebooks/Stormer-Verlet.ipynb [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "932f3d29",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Set up required packages\n",
+    "using Pkg\n",
+    "Pkg.activate(@__DIR__)\n",
+    "Pkg.instantiate()"
```
Comment (ranocha) [inline notebooks/Stormer-Verlet.ipynb:13]: We should also include the `Project.toml` and `Manifest.toml` files to ensure reproducibility with exactly the same package versions.

File: notebooks/Stormer-Verlet.ipynb [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+    "using Latexify\n",
+    "using LinearAlgebra"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f7593a1e",
+   "metadata": {},
+   "source": [
+    "Here we study the Stormer-Verlet method applied to the Kepler Problem:"
```
Comment (ranocha) [inline notebooks/Stormer-Verlet.ipynb:35]: I would feel slightly better if you could search & replace Stormer → Störmer :wink:

**Review by ranocha — COMMENTED** [review]
Thanks for this! My basic question is whether we should extend #3, this PR, or merge them?

**Comment by ketch** [comment]
> Thanks for this! My basic question is whether we should extend #3, this PR, or merge them?

Oops, I forgot about that.  I'll just close this and add code to that.

**Pull request closed without merging**